### DISCUSSION TITLE
Mystery text appears on "Your To Do" Page

### DISCUSSION THREAD

**Discussion by Flange0323** [Leantime/leantime#1527] [Q&A]
For example:

To-Dos this Week (%1$s) (4)

Can anyone tell me why I'm getting the "(%1$s) "

Thanks in advance

**Comment by revuponline** [comment]
We are also seeing this.

**Comment by marcelfolaron** [comment]
This is a replacement string that seems to be in an old language file.
Which language are you using?

**Reply by revuponline** [reply]
I was using en-US, I updated to .15 from .14 and the issue appears to have gone. I have also switched to en-GB now.
